### ISSUE TITLE
Is it banable?

### ISSUE THREAD

**Issue by BardiaAbedini** [drknzz/auto-lingo#32]
Can you get banned using an xp bot?
I really dont wanna lose my account

**Comment by Adadeane** [comment]
Obviously. Seems like you don't care about your account if you are using this.
